Doc/library/asyncio-task.rst

```diff
@@ -842,6 +842,12 @@ Task Object
       The *file* argument is an I/O stream to which the output
       is written; by default output is written to :data:`sys.stderr`.
 
+   .. method:: get_coro()
+
+      Return the coroutine object that was passed to the constructor.
```

Comment:
```suggestion
      Return the coroutine object wrapped by the :class:`Task`.
```
@1st1 like this?

Reply:
Yeah, I think it's great.  If you have time just merge Alex's PR and we can fix this ourselves. Up to you.

Reply:
I changed the wording now as requested.

Reply:
Great!